src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonStringEnumConverter.cs

```diff
@@ -6,6 +6,61 @@
 
 namespace System.Text.Json.Serialization
 {
+    /// <summary>
+    /// Converter to convert enums to and from strings.
+    /// </summary>
+    /// <remarks>
+    /// Reading is case insensitive, writing can be customized via a <see cref="JsonNamingPolicy" />.
+    /// </remarks>
+    /// <typeparam name="TEnum">The enum type that this converter targets.</typeparam>
+    public class JsonStringEnumConverter<TEnum> : JsonConverterFactory
```

Comment:
@eerhardt This was brought up in API review, do we need to annotate `TEnum` with `DynamicallyAccessedMembers` in case the trimmer removes enum values?

Reply:
Trimming never removes enum values. It cannot be done safely because `ToString` on enums relies on them - as soon as we see the enum getting boxed or it's typeof obtained, we keep all the fields. So this is fine without annotation - I assume there are no static analysis warnings either.